app/controllers/chores_controller.rb

```diff
@@ -0,0 +1,37 @@
+class ChoresController < ApplicationController
+  def create
+    begin
+      chore = Chore.create(chore_params)
```

Comment:
Since the hideout_id won't be delivered straight from the frontend, it'll be added by the backend in the params.payload object, reference `params[:payload]` first before accessing the hideout_id. Haven't implemented it yet, but I'll get it done tonight and push for you to test with tomorrow.

Reply:
added that as a comment for now